Source/WebKit/UIProcess/API/wpe/WPEWebViewPlatform.cpp

```diff
@@ -360,6 +224,50 @@ void ViewPlatform::updateDisplayID()
     m_pageProxy->windowScreenDidChange(m_displayID);
 }
 
+bool ViewPlatform::activityStateChanged(WebCore::ActivityState state, bool value)
+{
+    OptionSet<WebCore::ActivityState> flagsToUpdate { state };
```

Comment:
You can omit that local variable and just use "{ state }" below, when calling activityStateDidChange(), no?

Reply:
Yes, I think so